build.sbt

```diff
@@ -108,7 +111,7 @@ lazy val `support-workers` = (project in file("support-workers"))
     integrationTestSettings,
     libraryDependencies ++= commonDependencies
   ).dependsOn(`support-services`, `support-models` % "test->test;it->test;compile->compile", `support-config`, `support-internationalisation`)
-  .aggregate(`support-services`, `support-models`, `support-config`, `support-internationalisation`, `stripe-intent`, `support-redemptiondb`)
+  .aggregate(`support-services`, `support-models`, `support-config`, `support-internationalisation`, `stripe-intent`)
```

Comment:
the DB was never related to support workers, so removed it from the build definition.  This now happens in the build-tc script of support-workers-mono build.
So effectively it happens in the same place, but only in Teamcity.